src/load-nyc-config-sync.js

```diff
@@ -2,13 +2,16 @@
 'use strict'
 
 const { loadNycConfig } = require('@istanbuljs/load-nyc-config')
+const isElectron = process.versions && process.versions.electron
```

Comment:
I think this is unnecessary, we can/should do `process.exit(0)` unconditionally.